helpers/tcp_client.py

```diff
@@ -0,0 +1,209 @@
+        conf.L3socket = L3RawSocket
+
+    @classmethod
+    def rollback(cls) -> None:
+        conf.L3socket = None
+
+
+class ScapyTCPHandshakeResetFix(ScapyFix):
+    """
+    This fix creates rules in iptables to ignore kernel RESET answer
+    because of scapy uses user space to send packages.
+
+    https://scapy.readthedocs.io/en/latest/troubleshooting.html#my-tcp-connections-are-reset-by-scapy-or-by-my-kernel
+    """
+
+    @classmethod
+    def apply(cls) -> None:
+        cmd = [
+            "iptables",
+            "-A",
+            "OUTPUT",
+            "-p",
+            "tcp",
+            "--tcp-flags",
+            "RST",
+            "RST",
+            "--sport",
+            "55555",
+            "-j",
+            "DROP",
+        ]
+        result = subprocess.run(cmd, capture_output=True, text=True)
+        assert result.stderr == ""
+
+    @classmethod
+    def rollback(cls) -> None:
+        cmd = [
+            "iptables",
+            "-D",
+            "OUTPUT",
+            "-p",
+            "tcp",
+            "--tcp-flags",
+            "RST",
+            "RST",
+            "--sport",
+            "55555",
+            "-j",
+            "DROP",
+        ]
+        result = subprocess.run(cmd, capture_output=True, text=True)
+        assert result.stderr == ""
+
+
+class SimpleTCPClient:
+    def __init__(
+        self,
+        destination_host: str,
+        destination_port: int,
+        source_host: str = "127.0.0.1",
+        source_port: int = 55555,
+        timeout: int = 5,
+    ):
+        self.source_host = source_host
+        self.source_port = source_port
+        self.destination_host = destination_host
+        self.destination_port = destination_port
+        self.timeout = timeout
+
+        self.last_response: Optional[IP] = None
+        self.last_request: Optional[IP] = None
+
+        self.fixes = []
+
+    def fixes_install(self, *args: Type[ScapyFix]) -> None:
+        """
+        Apply some fixes for scapy
+        """
+        self.fixes = args
+        list(map(lambda item: item.apply(), self.fixes))
+
+    def fixes_rollback(self) -> None:
+        """
+        Rollback all installed fixes
+        """
+        list(map(lambda item: item.rollback(), self.fixes))
+
+    @contextlib.contextmanager
+    def fixes_context(self, *args: Type[ScapyFix]):
+        """
+        Apply scapy fixes and auto rollback
+        """
+        self.fixes_install(*args)
+        yield
+        self.fixes_rollback()
+
+    @staticmethod
+    def __build_packet(
+        flags: str,
+        src_host: str,
+        src_port: int,
+        dst_host: str,
+        dst_port: int,
+        seq: int = 0,
+        ack: int = 0,
+        data: bytes = None,
+    ) -> IP:
+        """
+        TCP request builder
+        """
+        req = IP(src=src_host, dst=dst_host) / TCP(
+            seq=seq,
+            ack=ack,
+            sport=src_port,
+            dport=dst_port,
+            window=65495,
```

Comment:
You should not use such magic numbers. It is better to add a reference to the value from the system or library